packages/eslint-plugin-pf-codemods/src/rules/v6/tokensWarn/tokens-warn.md

```diff
@@ -0,0 +1,17 @@
+### tokens-warn
+
+We have updated our CSS tokens. About half of our tokens have been replaced with newer ones. To find a suitable replacement token, check our new documentation https://staging-v6.patternfly.org/tokens/all-patternfly-tokens.
```

Comment:
Let's wrap the link in some link text for this MD file; we can keep the url as-is in the message sent to the terminal

```suggestion
We have updated our CSS tokens. About half of our tokens have been replaced with newer ones. To find a suitable replacement token, check our [v6 token documentation](https://staging-v6.patternfly.org/tokens/all-patternfly-tokens).
```